docs/index.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+Sophia is a language in the ML family designed for smart contract development. It is strongly typed and has
```

Comment:
```suggestion
Sophia is a functional language designed for smart contract development. It is strongly typed and has
```

Reply:
If you know ML (or another language in its family) you will recognize some syntax regardless, and if you don't, the "ML family" isn't going to provide you any insight 😅